hyperon_das_atomdb/utils/expression_hasher.py

```diff
@@ -123,7 +123,7 @@ class StringExpressionHasher:  # TODO(angelo,andre): remove this class? it's not
     """Utility class for generating string representations of expression hashes."""
```

Comment:
StringExpressionHasher is meant to be a hasher that generates human-readable unique strings. Something like the way we usually write nodes and links like "<concept: human>" etc. It's supposed to be used in tests, to make debugging easier but I suppose it's not used anymore. Anyway we get used to the regular hashes at the point of recognizing some of the more common ones :-P so I believe this class can be safely removed.

Reply:
I noticed it was not being used but was not sure if it could be safely removed. Will do it.